src/SurveyDesign.jl

```diff
@@ -157,4 +157,54 @@ struct ReplicateDesign <: AbstractSurveyDesign
     allprobs::Symbol # Right now only singlestage approx supported
     pps::Bool
     replicates::UInt
+    replicate_weights::Vector{Symbol}
+
+    # default constructor
```

Comment:
No, I don't think a default constructor is necessary. In our case, I had to add it because if we're creating a `ReplicateDesign` using functions like `bootweights` (or other similar functions, if we're planning to add them), we'll have to have a default constructor to just initialize the fields of the object.